api-reference/50 Common/Object Structures/FileSystemItem/dataItem.md

```diff
@@ -4,7 +4,6 @@ type: Object
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+The file system data object that stores information about the file system item (name, size, modification date, etc.).
```

Comment:
What is the object structure exactly? Which fields store what?

Reply:
This structure depends on file provider type. In a custom provider, a user implement the structure as he wishes.